simple/schema.py

```diff
@@ -268,3 +270,41 @@ class ModeledParameters(Base):
     unit = Column(String(20))
     comments = Column(String(1000))
     reference = Column(String(30), ForeignKey('Publications.reference', onupdate='cascade'), primary_key=True)
+
+
+# -------------------------------------------------------------------------------------------------------------------
+# Views
+
+ParallaxView = view(
+    "ParallaxView",
+    Base.metadata,
+    sa.select(
+        Parallaxes.source.label('source'),
+        Parallaxes.parallax.label('parallax'),
+        Parallaxes.parallax_error.label('parallax_error'),
+        (1000./Parallaxes.parallax).label('distance'),  # distance in parsecs
```

Comment:
I'd check this, remember we have some weird parallaxes in the database like negative values. Can you do a preliminary cut to get acceptable parallaxes?

Reply:
There are constraints to ensure only positive, non-zero, adopted parallaxes are used for this. So it should never break the calculation, though you may be missing entries if adopted is not properly set.